packages/engine/paima-sm/src/types.ts

```diff
@@ -249,6 +282,7 @@ export enum CdeEntryTypeName {
   ERC20Deposit = 'erc20-deposit',
   ERC721 = 'erc721',
   ERC6551Registry = 'erc6551-registry',
+  InverseAppProjected1155 = 'erc1155-app',
```

Comment:
Any reason to call `erc1155-app` instead of just `erc1155` ?

Reply:
I'm imagining that the IInverseBaseProjected1155 contract might be used later and could then be named `erc1155-base`, but maybe that's something that (like `erc721`) can be detected from the contract on startup.

Reply:
The most important event is transfers which are available on all ERC-1155, not specific to IInverseAppProjected1155, so in supporting that I've changed this to `erc1155` too.